CONTRIBUTING.md

```diff
@@ -0,0 +1,13 @@
+#Contributing
+
+Your pull requests will be welcome; however, they may not be accepted for various reasons.
+
+##Guidelines for pull requests:
+
+1. Respect Notepad++ coding style.
+2. Make a single change per commit.
+3. Make your modification compact - don't reformat source code in your request. It makes code review more difficult.
+
```

Comment:
*"This pull request can be automatically merged by project collaborators"* (the golden message)